.github/workflows/publish.yml

```diff
@@ -17,7 +17,8 @@ jobs:
     uses: wetransform/gha-workflows/.github/workflows/gradle-library.yml@master
```

Comment:
Please also use this as opportunity to reference a specific version of the workflow repo instead of the `master` branch. Thank you!

Reply:
Ok, I pinned the references to the latest gha-workflows commit.

I assume that an alternative to adapting the parameters would be to just use an older version of the workflow. Do you think that would be the better approach here?

Reply:
> I assume that an alternative to adapting the parameters would be to just use an older version of the workflow. Do you think that would be the better approach here?

Yes, using version 1 instead of 2 would resolve the issue as well, but it makes sense to do the update if we want to use the release workflow in the future.